resources/measures/advanced_rtu_control/tests/advanced_rtu_control_test.rb

```diff
@@ -0,0 +1,203 @@
+    if !File.exist?(run_dir(test_name))
+      FileUtils.mkdir_p(run_dir(test_name))
+    end
+    assert(File.exist?(run_dir(test_name)))
+
+    # change into run directory for tests
+    start_dir = Dir.pwd
+    Dir.chdir run_dir(test_name)
+
+    # remove prior runs if they exist
+    if File.exist?(model_output_path(test_name))
+      FileUtils.rm(model_output_path(test_name))
+    end
+    if File.exist?(report_path(test_name))
+      FileUtils.rm(report_path(test_name))
+    end
+
+    # copy the osm and epw to the test directory
+    new_osm_path = "#{run_dir(test_name)}/#{File.basename(osm_path)}"
+	puts("osm path" + osm_path)
+	puts("new osm path" + new_osm_path)
+    FileUtils.cp(osm_path, new_osm_path)
+    new_epw_path = "#{run_dir(test_name)}/#{File.basename(epw_path)}"
+    FileUtils.cp(epw_path, new_epw_path)
+    # create an instance of a runner
+    runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new)
+
+    # load the test model
+    if model.nil?
+      model = load_model(new_osm_path)
+    end
+
+    # set model weather file
+    epw_file = OpenStudio::EpwFile.new(OpenStudio::Path.new(new_epw_path))
+    OpenStudio::Model::WeatherFile.setWeatherFile(model, epw_file)
+    assert(model.weatherFile.is_initialized)
+
+    # run the measure
+    puts "\nAPPLYING MEASURE..."
+    measure.run(model, runner, argument_map)
+    result = runner.result
+    result_success = result.value.valueName == 'Success'
+
+    # Show the output
+    show_output(result)
+
+    # Save model
+    model.save(model_output_path(test_name), true)
+
+    if run_model && result_success
+      puts "\nRUNNING MODEL..."
+
+      std = Standard.build('90.1-2013')
+      std.model_run_simulation_and_log_errors(model, run_dir(test_name))
+
+      # Check that the model ran successfully
+      assert(File.exist?(sql_path(test_name)))
+    end
+
+    # change back directory
+    Dir.chdir(start_dir)
+
+    return result
+  end
+  
+  #test for a particula rbuilding type 
+  
+  def test_retail_PSZ_gas_5a
+   
+   # this makes sure measure registers an na for non applicable model
+    osm_name = '361_Retail_PSZ_Gas_5a_added_output.osm'
+    epw_name = 'USA_NE_Kearney.Muni.AWOS.725526_TMY3.epw'
+
+    osm_path = model_input_path(osm_name)
+	osm_path_output = model_output_path(osm_name)
+    epw_path = epw_input_path(epw_name)
+
+    # Create an instance of the measure
+    measure = AdvancedRTUControl.new
+
+    # Load the model; only used here for populating arguments
+    #model = load_model(osm_path)
+	puts ("#{File.dirname(__FILE__)}" + "//models//" + "#{osm_name}") 
+	model = load_model(osm_path)
+    #osm_path = "C://Users//aallen//Documents//GitHub//ComStock//resources//measures//advanced_rtu_control//tests//models//361_Retail_PSZ_Gas_5a_added_output.osm"
+	arguments = measure.arguments(model)
+    argument_map = OpenStudio::Measure::OSArgumentMap.new
+	
+	# Apply the measure to the model and optionally run the model
```

Comment:
More indent fixes - carry this comment forward to properly align indents.

Reply:
Yes, tried to do this throughout. Thanks!